cachito/web/api_v1.py

```diff
@@ -29,12 +33,30 @@ def get_requests():
         value exceeds configuration's CACHITO_MAX_PER_PAGE.
     :rtype: flask.Response
     """
-    # Default versbose flag to False
+    # Check if the user is filtering requests by state
+    state = flask.request.args.get('state')
+    # Default verbose flag to False
     verbose = str_to_bool(flask.request.args.get('verbose', False))
     max_per_page = flask.current_app.config['CACHITO_MAX_PER_PAGE']
     # The call to `paginate` will inspect the current HTTP request for the
     # pagination parameters `page` and `per_page`.
-    pagination_query = Request.query.paginate(max_per_page=max_per_page)
+    query = Request.query
+    if state:
+        if state not in RequestStateMapping.get_state_names():
+            raise ValidationError(
+                f'{state} is not a valid request state. Valid states are: '
+                f'{":".join(RequestStateMapping.get_state_names())}'
```

Comment:
```suggestion
                ':'.join(RequestStateMapping.get_state_names())
```

Reply:
I changed the quotes, but also put it into a separate variable because I think it makes it more readable.